lib/ansible/utils/module_docs_fragments/kubevirt_vm_options.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+
+class ModuleDocFragment(object):
+
+    # Standard oVirt documentation fragment
+    DOCUMENTATION = '''
+options:
+    wait:
```

Comment:
This is already defined in kubevirt_common_options.

Reply:
ah, right, thanks